sdk/maps/azure-maps-traffic/src/main/java/com/azure/maps/traffic/models/TrafficIncidentDetailOptions.java

```diff
@@ -0,0 +1,202 @@
+package com.azure.maps.traffic.models;
+
+import com.azure.core.models.GeoBoundingBox;
+
+public final class TrafficIncidentDetailOptions {
+    private ResponseFormat format;
+    private IncidentDetailStyle style;
+    private GeoBoundingBox boundingbox;
+    private int boundingZoom;
```

Comment:
for all options, can you please use Integer instead of int? just to standardize everything